compositor_render/src/transformations/layout.rs

```diff
@@ -161,3 +161,24 @@ impl LayoutNode {
             .render(ctx.wgpu_ctx, self.params.bind_group(), &textures, target);
     }
 }
+
+impl NestedLayout {
+    /// NestedLayout that can won't ever be rendered. It's intended to optimized out
+    /// in the flattening process. Its only propose is to keep track of child nodes that are not
+    /// currently used, so the index offset can be calculated correctly.
```

Comment:
```suggestion
    /// NestedLayout that won't ever be rendered. It's intended to be optimized out
    /// in the flattening process. Its only purpose is to keep track of child nodes that are not
    /// currently used so that the index offset can be calculated correctly.
```